tap_hubspot/hubspot.py

```diff
@@ -297,9 +297,10 @@ def get_owners(self):
     def get_archived(self, object_type: str):
         path = f"/crm/v3/objects/{object_type}"
         data_field = "results"
-        replication_path = ["archivedAt"]
+        filter_key = "archivedAt"
+        replication_path = [filter_key]
         offset_key = "after"
-        params = {"limit": 100, "archived": True}
+        params = {"limit": 100, "archived": True, "properties": filter_key}
```

Comment:
the important thing here, apart from the filter_key - which I suspect is there regardless, is the contact/company/deals. What I want to ensure is that we are getting the minimal amount of properties, to make the data as small as possible because we only need `archivedAt` and `id` for the rest of the pipeline to figure out which of the deals/contacts are now deleted.

Reply:
since the fields I am talking about are _probably_ all there regardless, check if we can get away with an empty properties field (ie. `=[]`). When we left out the properties field instead of adding two fields in the search endpoint, it gave us waaay too many properties (which I suspect are just default).

Reply:
Ok so if we pass in hs_object_id we get hs_object_id, lastmodifieddate and createdate back.
If we pass an empty list we get. createdate, email, firstname, hs_object_id, lastmodfieddate, lastname.
So I guess we just pass in the hs_object_id then? Then we get as less as possible.